src/store.js

```diff
@@ -63,8 +65,8 @@ function handleRDFType (formula, subj, pred, obj, why) {
   }
   return done // statement given is not needed if true
 }
-/**
- * IndexedFormula
+/*
+ * PanicPanic
```

Comment:
? 😄

Reply:
That was a debug trace for trying to understand  jsdoc

Reply:
That was a debug trace for trying to understand  jsdoc  Good catch